packages/cnpj-gen/src/CnpjGeneratorOptions.php

```
@@ -2,67 +2,407 @@
+     * @throws CnpjGeneratorOptionTypeInvalidException If the value is not a
+     *   valid type.
+     */
+    private function parseCnpjType(string $optionName, mixed $value): CnpjType
     {
-        return $this->format;
+        if ($value instanceof CnpjType) {
+            return $value;
+        }
+
+        if (is_string($value)) {
+            $cnpjType = CnpjType::tryFrom($value);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Lacus\BrUtils\Cnpj\Enums\CnpjType' in method 'parseCnpjType'. (undefined)

(StaticAccess)